tests/storages_tests/test_storages.py

```diff
@@ -968,6 +975,8 @@ def _setup_studies(
     n_trial: int,
     seed: int,
     direction: Optional[StudyDirection] = None,
+    use_study_user_attrs: bool = False,
+    use_study_system_attrs: bool = False,
```

Comment:
How about simply always using user and system attributes, similar to how we generate trials? Or do we want to be able to configure this?

Reply:
+1 to use user and system attributes by default. I've confirmed that the any existing tests do not fail if we do this.

Reply:
Thank you for your suggestion and confirmation. I'll update the test case.

Reply:
Fixed in https://github.com/optuna/optuna/pull/3352/commits/50bf73450405395b4b7fc6aeea56152d9003f9b5. PTAL!